Cargo.toml

```diff
@@ -13,7 +13,7 @@ default = ["pyo3/num-bigint", "pyo3/auto-initialize"]
 
 [dependencies]
 pyo3 = { version = "0.16.5" }
-cairo-rs = { git = "https://github.com/lambdaclass/cairo-rs.git", rev = "012681e82f0617b723fe3433723a95a64eafafc9" }
+cairo-rs = { git = "https://github.com/lambdaclass/cairo-rs.git", rev = "b6baeccb0e38266ec74b500d7ebe64ac8e8a1e0a" }
```

Comment:
TODO
Update commit once [cairo-rs PR ](https://github.com/lambdaclass/cairo-rs/pull/634) is merged

Reply:
done